checks/check-app-health.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"net/http"
+	"io/ioutil"
+    "strconv"
+    "log"
+
+	maps "github.com/ashwanthkumar/golang-utils/maps"
+	"github.com/gambol99/go-marathon"
+)
+
+type AppHealth struct{}
+
+func (h *AppHealth) Name() string {
+	return "apphealth"
+}
+
+// Checks App health status of App Endpoint by reading router.hosts 
+//  and Check for 200 status code
+func (h *AppHealth) Check(app marathon.Application) AppCheck {
+	host := maps.GetString(app.Labels, "router.hosts", "notDefined")
+    host_path  := app.HealthChecks[0].Path
```

Comment:
We should be checking for only HTTP type health sppec.

Reply:
This is the first step, now we check for HTTP and if so carry on. If not HTTP, we give a mild warning and message...any other suggestions, commit below
https://github.com/ashwanthkumar/marathon-alerts/pull/35/commits/408f173f47b17f6c30e16d13cc58b1e5b3f1dbb2